src/WebJobs.Extensions.DurableTask/DurableOrchestrationClientBase.cs

```diff
@@ -176,5 +178,14 @@ public virtual Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId
         /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
         /// <returns>Returns a task which completes when the status has been fetched.</returns>
         public abstract Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId, bool showHistory, bool showHistoryOutput);
+
+        /// <summary>
+        /// Gets all the status of the orchestration instances.
+        /// </summary>
+        /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
+        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
+        /// <param name="cancellationToken">Cancel your request by this token</param>
```

Comment:
I would phrase this token a little differently to match the passive wording style of the other parameter descriptions. For example:
> Cancellation token that can be used to cancel the status query operation.

Reply:
Thanks. Done it.